src/display.rs

```diff
@@ -368,9 +375,7 @@ impl Display {
                 let window_focused = self.window.is_focused;
                 self.renderer.with_api(config, &size_info, visual_bell_intensity, |mut api| {
                     // Clear screen to update whole background with new color
-                    if background_color_changed {
-                        api.clear(background_color);
-                    }
+                    api.clear(background_color);
```

Comment:
Since the background is cleared previously, it should not be required to have this `api.clear` call anymore.